templates/virtualization-controller/vmcpu-legacy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: virtualization.deckhouse.io/v1alpha2
+kind: VirtualMachineCPUModel
+metadata:
+  name: legacy
+spec:
+  type: Model
+  model: "Nehalem"
```

Comment:
perhaps we want to create all available cpu models at once.  https://github.com/libvirt/libvirt/blob/master/src/cpu_map/index.xml

Reply:
It's necessary to create only Nehalem model as preset (in accordance with rnd/api/cpu-model requirements)